misaligned/misaligned.cs

```diff
@@ -1,21 +1,28 @@
 ﻿using System;
 using System.Diagnostics;
 
-namespace MisalignedSpace {
-    class Misaligned {
-        static int printColorMap() {
-            string[] majorColors = {"White", "Red", "Black", "Yellow", "Violet"};
-            string[] minorColors = {"Blue", "Orange", "Green", "Brown", "Slate"};
-            int i = 0, j = 0;
-            for(i = 0; i < 5; i++) {
-                for(j = 0; j < 5; j++) {
-                    Console.WriteLine("{0} | {1} | {2}", i * 5 + j, majorColors[i], minorColors[i]);
+namespace MisalignedSpace
+{
+    class Misaligned
+    {
+        static int printColorMap(string[] majorColors, string[] minorColors)
+        {
+
+            int i, j = 0;
+            for (i = 0; i < 5; i++)
+            {
+                for (j = 0; j < 5; j++)
+                {
+                    Console.WriteLine("{0} | {1} | {2}", i * 5 + j, majorColors[i], minorColors[j]);

```

Comment:
5 is a number that can be misinterpreted in future. Can you please think if the code works if major and minor color length is different than 5,5 ?